### ISSUE TITLE
Resource changes broke js proxy

### ISSUE THREAD

**Issue by davidfowl** [aspnet/SignalR-Server#96]
/cc @victorhurdugaci you forgot to update here:

```
An unhandled exception occurred while processing the request.

ArgumentNullException: Argument cannot be null.
Parameter name: stream
System.IO.StreamReader.Initialize (System.IO.Stream stream, System.Text.Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize) [0x00000] in <filename unknown>, line 0

Stack Query Cookies Headers Environment
ArgumentNullException: Argument cannot be null. Parameter name: stream
System.IO.StreamReader.Initialize (System.IO.Stream stream, System.Text.Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize) [0x00000] in <filename unknown>
System.IO.StreamReader..ctor (System.IO.Stream stream, System.Text.Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize, Boolean leaveOpen) [0x00000] in <filename unknown>
System.IO.StreamReader..ctor (System.IO.Stream stream, System.Text.Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize) [0x00000] in <filename unknown>
System.IO.StreamReader..ctor (System.IO.Stream stream) [0x00000] in <filename unknown>
in (wrapper remoting-invoke-with-check) System.IO.StreamReader
Microsoft.AspNet.SignalR.Hubs.DefaultJavaScriptProxyGenerator.GetTemplateFromResource () [0x00000] in <filename unknown>
System.Lazy`1[System.String].InitValue () [0x00000] in <filename unknown>
--- End of stack trace from previous location where exception was thrown ---
```

**Comment by rajajhansi** [comment]
@victorhurdugaci, I get the same exception when I try to include the <MyUrl>/signalr/js in <script> tag. When I look at the aspnextvnext MyGet feed, it looks like it was last updated 4 days ago and the version is  Microsoft.AspNet.SignalR.Server	v3.0.0-beta5-12256.

**Comment by victorhurdugaci** [comment]
Should be fixed by https://github.com/aspnet/SignalR-Server/pull/97

**Comment by rajajhansi** [comment]
Fix #97 worked !!! Thanks.
